lib/autoupdate/agent/telemetry.go

```diff
@@ -31,6 +31,13 @@ import (
 // The binary is considered managed if it lives under /opt/teleport, but not within the package
 // path at /opt/teleport/system.
 func IsManagedByUpdater() (bool, error) {
```

Comment:
nit: not related to this PR changes, I would probably move `IsManagedByUpdater` to `utils` in such case we don't compile and link agent auto-update logic to teleport binary